docs/credentials/credential_plugins.md

```diff
@@ -117,12 +117,22 @@ setuptools.setup(
     entry_points = {
         ...,
         'awx.credential_plugins': [
-            'fancy_plugin = awx.main.credential_plugins.fancy:some_fancy_plugin',
+            'fancy = my_fancy_project.my_fancy_module:some_fancy_plugin',  # some_fancy_plugin = CredentialPlugin(...)
         ]
     }
 )
 ```
 
+Enable Custom Credential Plugins
+--------------------------------
+
+Once the Python package has been installed, `awx-manage` must be used in order
```

Comment:
What we've *been* doing is introducing a migration with each new plugin that *calls* this function and auto-detects and registers it.

Reply:
That doesn't apply to custom credential plugins, isn't it ?